From b7e918a79e663a76817f529c77e984d903fd9dbd Mon Sep 17 00:00:00 2001
From: Kevin Hilman <khilman@deeprootsystems.com>
Date: Fri, 13 Feb 2009 11:03:17 -0800
Subject: [PATCH 352/364] OMAP3: PM: minor cleanup of PRM register definitions

The prm-regbits-*.h files should have register and field definitions
but not any register value definitions.  Some are no longer needed
after the PRM setup changes, and others have been moved to pm34xx.c.

Signed-off-by: Kevin Hilman <khilman@deeprootsystems.com>
(cherry picked from commit 1d6d449526ffa462744f2c218eee790d316de4af)
(cherry picked from commit 0266b090de0ff814ad0a556bca4979c25179978a)
---
 arch/arm/mach-omap2/pm34xx.c           |   12 ++++++++++++
 arch/arm/mach-omap2/prm-regbits-34xx.h |   27 ++++-----------------------
 2 files changed, 16 insertions(+), 23 deletions(-)

diff --git a/arch/arm/mach-omap2/pm34xx.c b/arch/arm/mach-omap2/pm34xx.c
index b136383..70bc793 100644
--- a/arch/arm/mach-omap2/pm34xx.c
+++ b/arch/arm/mach-omap2/pm34xx.c
@@ -1053,6 +1053,18 @@ err2:
 	return ret;
 }
 
+/* PRM_VC_CMD_VAL_0 specific bits */
+#define OMAP3430_VC_CMD_VAL0_ON				(0x3 << 4)
+#define OMAP3430_VC_CMD_VAL0_ONLP			(0x3 << 3)
+#define OMAP3430_VC_CMD_VAL0_RET			(0x3 << 3)
+#define OMAP3430_VC_CMD_VAL0_OFF			(0x3 << 4)
+
+/* PRM_VC_CMD_VAL_1 specific bits */
+#define OMAP3430_VC_CMD_VAL1_ON				(0xB << 2)
+#define OMAP3430_VC_CMD_VAL1_ONLP			(0x3 << 3)
+#define OMAP3430_VC_CMD_VAL1_RET			(0x3 << 3)
+#define OMAP3430_VC_CMD_VAL1_OFF			(0xB << 2)
+
 static void __init configure_vc(void)
 {
 
diff --git a/arch/arm/mach-omap2/prm-regbits-34xx.h b/arch/arm/mach-omap2/prm-regbits-34xx.h
index d73eee8..cb648f9 100644
--- a/arch/arm/mach-omap2/prm-regbits-34xx.h
+++ b/arch/arm/mach-omap2/prm-regbits-34xx.h
@@ -454,25 +454,13 @@
 
 /* PRM_VC_CMD_VAL */
 #define OMAP3430_VC_CMD_ON_SHIFT			24
-#define OMAP3430_VC_CMD_ON_MASK				(0xFF << 24)
+#define OMAP3430_VC_CMD_ON_MASK				(0xff << 24)
 #define OMAP3430_VC_CMD_ONLP_SHIFT			16
-#define OMAP3430_VC_CMD_ONLP_MASK			(0xFF << 16)
+#define OMAP3430_VC_CMD_ONLP_MASK			(0xff << 16)
 #define OMAP3430_VC_CMD_RET_SHIFT			8
-#define OMAP3430_VC_CMD_RET_MASK			(0xFF << 8)
+#define OMAP3430_VC_CMD_RET_MASK			(0xff << 8)
 #define OMAP3430_VC_CMD_OFF_SHIFT			0
-#define OMAP3430_VC_CMD_OFF_MASK			(0xFF << 0)
-
-/* PRM_VC_CMD_VAL_0 specific bits */
-#define OMAP3430_VC_CMD_VAL0_ON				(0x3 << 4)
-#define OMAP3430_VC_CMD_VAL0_ONLP			(0x3 << 3)
-#define OMAP3430_VC_CMD_VAL0_RET			(0x3 << 3)
-#define OMAP3430_VC_CMD_VAL0_OFF			(0x3 << 4)
-
-/* PRM_VC_CMD_VAL_1 specific bits */
-#define OMAP3430_VC_CMD_VAL1_ON				(0xB << 2)
-#define OMAP3430_VC_CMD_VAL1_ONLP			(0x3 << 3)
-#define OMAP3430_VC_CMD_VAL1_RET			(0x3 << 3)
-#define OMAP3430_VC_CMD_VAL1_OFF			(0xB << 2)
+#define OMAP3430_VC_CMD_OFF_MASK			(0xff << 0)
 
 /* PRM_VC_CH_CONF */
 #define OMAP3430_CMD1					(1 << 20)
@@ -531,13 +519,6 @@
 #define OMAP3430_AUTO_RET				(1 << 1)
 #define OMAP3430_AUTO_SLEEP				(1 << 0)
 
-/* Constants to define setup durations */
-#define OMAP3430_CLKSETUP_DURATION			0xff
-#define OMAP3430_VOLTSETUP_TIME2			0xfff
-#define OMAP3430_VOLTSETUP_TIME1			0xfff
-#define OMAP3430_VOLTOFFSET_DURATION			0xff
-#define OMAP3430_VOLTSETUP2_DURATION			0xff
-
 /* PRM_SRAM_PCHARGE */
 #define OMAP3430_PCHARGE_TIME_SHIFT			0
 #define OMAP3430_PCHARGE_TIME_MASK			(0xff << 0)
-- 
1.5.6

